server/models/data/Timeslot.js

```diff
@@ -0,0 +1,12 @@
+const UserTypes = require('../../constants/userTypes.json');
+
+class Timeslot {
+
```

Comment:
Hi Peter,
To avoid unnecessarily long files, I would recommend removing this extra whitespace on line 4.

Reply:
I agree, done